### DISCUSSION TITLE
Activate HDR by httprequest

### DISCUSSION THREAD

**Discussion by tommyguuuun** [awawa-dev/HyperHDR#483] [Q&A] [answered]
Hey,
i really love the functionality of HyperHDR. One downside is, the inconvenience of activating hdr conversion by hand.

the most convinient solution would be to do a httprequest button on my iobroker, now i wonder if there‘s any command to do so?

**Comment by awawa-dev** [comment, marked as answer]
Hi
In "advanced->json api" tab you can create a remote request to enable/disable HDR.

**Reply by tommyguuuun** [reply]
Perfect! Thanks for your fast answer and your awesome work!
